viewing_party/party.py

```diff
@@ -1,23 +1,179 @@
+from collections import Counter
+from shutil import move
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {
+        "title":title,
+        "genre":genre,
+        "rating":rating
+    }
+
+    for val in movie_dict.values():
+        if not val:
+            return None
+
+    return movie_dict
+
+def add_to_watched(user, movie):
+
+    for key in user:
+        user[key].append(movie)

```

Comment:
This does accomplish the task to add a movie to user["watched"] and the tests pass! However, since you're iterating through all the keys in user, if user had "watched", "watch_list", "friends", "favorites" when add_to_watched() is called then you would be adding the movie to all those attributes. 

We only need to add the movie to "watched". You could do that by explicitly accessing watched and drop the for loop, something like: 
`user["watched"].append(movie)`